src/proxnut/proxnut.py

```diff
@@ -71,14 +74,19 @@ def validate(self):
 
         # Validate Proxmox configuration
         logger.info("Validating Proxmox configuration...")
-        if not self.target_machines:
-            raise ValidateError("No target machines configured")
-
-        if not self.proxmox_client.validate_target_nodes(self.target_machines):
+        if self.shutdown_all_nodes:
+            # If no target machines configured, validate that cluster has at least one node
+            logger.info("No target machines configured. Will shutdown all nodes in cluster at shutdown time.")
             available_nodes = self.proxmox_client.get_nodes()
-            raise ValidateError(
-                "Target nodes not found in Proxmox cluster", available_nodes
-            )
+            if not available_nodes:
+                raise ValidateError("No nodes found in Proxmox cluster")
```

Comment:
It is too much. No need to do logging such a obvious thing. Also, no node check is unnecessary.